### ISSUE TITLE
Crash Godot 2.1.4 onInputDeviceRemoved

### ISSUE THREAD

**Issue by megasoft78** [godotengine/godot#17987]
Hi guys,
I'm getting the following crash for my game from Google Developer Console:
```
java.lang.ArrayIndexOutOfBoundsException: 
  at java.util.ArrayList.remove (ArrayList.java:480)
  at org.godotengine.godot.GodotView.onInputDeviceRemoved (GodotView.java:236)
  at org.godotengine.godot.input.InputManagerV16$V16InputDeviceListener.onInputDeviceRemoved (InputManagerV16.java:70)
  at android.hardware.input.InputManager$InputDeviceListenerDelegate.handleMessage (InputManager.java:1451)
  at android.os.Handler.dispatchMessage (Handler.java:102)
  at android.os.Looper.loop (Looper.java:154)
  at android.app.ActivityThread.main (ActivityThread.java:6682)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:1520)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1410)
```

It looks like the device is trying to remove doesn't exists.

Any idea on how to solve it?

**Comment by Hinsbart** [comment]
possibly related to #17630

**Comment by akien-mga** [comment]
Fixed by #17999.
